app/assets/javascripts/application/suggest_user.js

```diff
@@ -0,0 +1,24 @@
+function suggestUser() {
+    var url = "/search/suggest_leader";
+    var params = "query=" + document.getElementsByName('group_leader_name')[0].value;
+    http = new XMLHttpRequest();
```

Comment:
Can you use `fetch` API here instead?

Reply:
I made an iteration with the fetch API. Is using catch and log the error a good pratice or should be removed?